Sources/NIOExtrasPerformanceTester/HTTP1PCAPPerformanceTests.swift

```diff
@@ -15,21 +15,27 @@
 import NIOCore
 import NIOExtras
 import Foundation
+import NIOConcurrencyHelpers
 
 class HTTP1ThreadedPCapPerformanceTest: HTTP1ThreadedPerformanceTest {
-    private class SinkHolder {
+    class SinkHolder {
```

Comment:
Did we need to make this non-`private`?

Reply:
Bump

Reply:
@Lukasa nope, this was to be able to extend it (could also have used `fileprivate` I guess, but I'll revert as we're no longer extending.